components/apprentice-qualities/apprentice-qualities.module.scss

```diff
@@ -11,16 +11,15 @@
   width: $component-width;
 
   &__main {
-    margin: auto;
+    @include horizontal-section-spacing();
+    box-sizing: border-box;
```

Comment:
This is looking great Alice, I think I figured out a simple way to stop the text from floating off to the right on larger screens.

Could we add this CSS to `&__main`?
```
    justify-content: right;
    max-width: $max-content-width + ($xlarge-screen-padding * 2);
    margin: auto;
```

☝️ Setting the `max-width` this way feels a bit hacky, but absent a bigger refactor to the DOM structure, it it's a simple way to account for the padding with `border-box` setting.